spec/customer_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'customer'
+
+describe Customer do
+
+  context '#menu' do
+
+    it 'can see the menu' do
+      allow(subject).to receive(:menu).and_return(:Burger)
+      expect(subject.menu).to eq :Burger
+    end
+  end
+
+  context '#place_order' do
+
+    before { subject.place_order :Burger }
+
+    it 'will raise an error when dish is not on the menu' do
+      expect { subject.place_order(:Duck) }.to raise_error 'Please select a dish from the menu.'
```

Comment:
Metrics/LineLength: Line is too long. [96/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)